howfast_apm/utils.py

```diff
@@ -1,22 +1,28 @@
 import fnmatch
+import re
 
 from typing import List
 
 
+def convert_endpoints(*blacklist):
```

Comment:
I think we can rename it to `compile_endpoints` to make it slightly more explicit
```suggestion
def compile_endpoints(*blacklist):
```

Reply:
yes